test/old-start-configs/run.sh

```diff
@@ -7,9 +7,6 @@ OS_ROOT=$(dirname "${BASH_SOURCE}")/../../..
 source "${OS_ROOT}/hack/util.sh"
 cd "${OS_ROOT}"
 
-ensure_iptables_or_die
```

Comment:
> Why doesn't test-end-to-end ensure iptables?

It's based on the old script.  Looking specifically, it doesn't the common method, but it embedded the logic.